utilities/openemr-cmd/openemr-cmd

```diff
@@ -321,125 +353,151 @@ then
+  echo "  cwo, change-webroot-openemr        Change webroot to be openemr"
   echo 'others:'
-	echo "  ev, ensure-version                 Upgrade OpenEMR from specified old version to current version"
-	echo "  el, enable-ldap                    Turn on support for LDAP - login credentials are admin:admin"
-	echo "  dld, disable-ldap                  Turn off support for LDAP - standard login credentials"
-	echo "  ec, encoding-collation             Change the database character set and collation"
-
-	exit $USAGE_EXIT_CODE
-elif [ "$FIRST_ARG" == '--version' ] || [ "$FIRST_ARG" == '-v' ]
-then
-	echo "openemr-cmd $VERSION"
-	exit $VERSION_EXIT_CODE
+  echo "  ev, ensure-version                 Upgrade OpenEMR from specified old version to current version"
+  echo "  el, enable-ldap                    Turn on support for LDAP - login credentials are admin:admin"
+  echo "  dld, disable-ldap                  Turn off support for LDAP - standard login credentials"
+  echo "  ec, encoding-collation             Change the database character set and collation"
+
+  exit "${USAGE_EXIT_CODE}"
+
+elif [[ "${FIRST_ARG}" == '--version' ]] || [[ "${FIRST_ARG}" == '-v' ]]; then
+	echo "openemr-cmd ${VERSION}"
+	exit "${VERSION_EXIT_CODE}"
 fi
 
 # Specify the docker id/name to execute the commands.
-#  default to insane dev docker, if exists
-CONTAINER_ID=$(docker ps | grep $INSANE_DEV_DOCKER | cut -f 1 -d " ")
-if [ -z $CONTAINER_ID ]
-then
-  # else, default to easy dev docker, if exists
-  CONTAINER_ID=$(docker ps | grep $EASY_DEV_DOCKER | cut -f 1 -d " ")
+# Default to insane dev docker, if exists, otherwise easy dev docker.
+# Try to find the insane dev docker ID directly
+if insane_id=$(docker ps --filter "name=${INSANE_DEV_DOCKER}" --format "{{.ID}}" | head -n 1); [[ -n "${insane_id}" ]]; then
+  CONTAINER_ID="${insane_id}"
+# Try to find the easy dev docker ID directly
+elif easy_id=$(docker ps --filter "name=${EASY_DEV_DOCKER}" --format "{{.ID}}" | head -n 1); [[ -n "${easy_id}" ]]; then
+  CONTAINER_ID="${easy_id}"
+else
+  # Neither container found running, set CONTAINER_ID to empty.
+  # The script's later logic handles the -d flag or errors if no container is specified.
+  CONTAINER_ID=""
 fi
+
 # override the default container setting if supplied via the -d parameter
 #  (also removing no longer needed parameters via shift)
 CHANGE_CONTAINER_CODE=25
-if [ "$FIRST_ARG" == '-d' ] && [ $# -ge 3 ]
-then
-	CONTAINER_ID=$2
-	FIRST_ARG=$3
+if [[ "${FIRST_ARG}" == '-d' ]] && [[ $# -ge 3 ]]; then
+	CONTAINER_ID="$2"
+	FIRST_ARG="$3"
 	shift 3
-elif [ "$FIRST_ARG" == '-d' ] && [ $# -lt 3 ]
-then
+elif [[ "${FIRST_ARG}" == '-d' ]] && [[ $# -lt 3 ]]; then
 	echo 'Please provide a docker id/name when using -d parameter.'
 	echo 'e.g. openemr-cmd -d docker_openemr-7-3-redis-312_1 dl'
-	exit $CHANGE_CONTAINER_CODE
+	exit "${CHANGE_CONTAINER_CODE}"
 else
   shift
 fi
```

Comment:
```suggestion
if [[ "${FIRST_ARG}" = '-d' ]]; then
  if (( $# < 3 )); then
    echo 'Please provide a docker id/name when using -d parameter.'
    echo 'e.g. openemr-cmd -d docker_openemr-7-3-redis-312_1 dl'
    exit "${CHANGE_CONTAINER_CODE}"
  fi
  CONTAINER_ID="$2"
  FIRST_ARG="$3"
  shift 3
else
  shift
fi
```

Reply:
Can you retry this suggestion, it is giving me an error "Applying suggestions on deleted lines is currently not supported. "